.goreleaser.yml

```diff
@@ -88,32 +99,37 @@ dockers:
       - scripts
       - web
       - widget
-  - dockerfile: cmd/scanner/Dockerfile
-    image_templates:
-      - "artifacthub/scanner:{{ .Tag }}"
-      - "artifacthub/scanner:latest"
-      - "public.ecr.aws/artifacthub/scanner:{{ .Tag }}"
-      - "public.ecr.aws/artifacthub/scanner:latest"
+  - id: scanner
+    dockerfile: cmd/scanner/Dockerfile
+    images:
+      - artifacthub/scanner
+      - public.ecr.aws/artifacthub/scanner
+    tags:
+      - "{{ .Tag }}"
+      - latest
     extra_files:
       - go.mod
       - go.sum
       - cmd/scanner
       - internal
-  - dockerfile: cmd/tracker/Dockerfile
-    image_templates:
-      - "artifacthub/tracker:{{ .Tag }}"
-      - "artifacthub/tracker:latest"
-      - "public.ecr.aws/artifacthub/tracker:{{ .Tag }}"
-      - "public.ecr.aws/artifacthub/tracker:latest"
+  - id: tracker
+    dockerfile: cmd/tracker/Dockerfile
+    images:
+      - artifacthub/tracker
+      - public.ecr.aws/artifacthub/tracker
+    tags:
+      - "{{ .Tag }}"
+      - latest
     extra_files:
       - go.mod
       - go.sum
       - cmd/tracker
       - internal
       - ml
 
-brews:
-  - description: Artifact Hub command line tool
+homebrew_casks:
```

Comment:
To make the transition to the cask easier for users I think we should:

- Make cask conflict with previous formula
- Tell macOS to [remove the quarantine bit from the binary](https://goreleaser.com/customization/homebrew_casks/#signing-and-notarizing)

Could you add both to this section?

The next time we create a new release, we'll also disable the formula.